app/_src/gateway/key-concepts/consumers.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Consumers
+concept_type: reference
+---
+
+## What is a consumer?
+
+A consumer typically refers to an entity that consumes or uses the APIs managed by {{site.base_gateway}}. 
+Consumers can be applications, services, or users who interact with your APIs. 
+Since they are not always human, {{site.base_gateway}} calls them consumers, as they "consume" the service.
+{{site.base_gateway}} allows you to define and manage consumers, apply access control policies, and monitor their usage of APIs.
+
+Consumers are essential for controlling access to your APIs, tracking usage, and ensuring security.
+They are identified by key authentication, OAuth, or other authentication and authorization mechanisms. 
+For example, adding a Basic Auth plugin to a service or route allows it to identify a consumer, or block access if credentials are bad.
```

Comment:
At first I thought linking to the plugin might be a good idea, but I think it might detract from what you are trying to teach.

Reply:
yep, the specific auth plugin is not important in this case, it could be any auth plugin. linking off to it would be an unnecessary distraction.